dashboard/app/design-lab/variants/VariantA.tsx

```
@@ -0,0 +1,251 @@
+            ARI <span style={{ color: "#268bd2" }}>↔</span> IVAN
+          </span>
+          <nav className="flex items-center gap-1">
+            {tabs.map((t) => (
+              <button key={t.id} onClick={() => setTab(t.id)}
+                className="px-3 py-1 rounded text-xs font-medium"
+                style={{
+                  backgroundColor: tab === t.id ? "#002b36" : "transparent",
+                  color: tab === t.id ? "#eee8d5" : "#586e75",
+                  border: tab === t.id ? "1px solid #586e75" : "1px solid transparent",
+                  cursor: "pointer",
+                }}>
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)